src/renderer/pages/common/SortsAndFilters.tsx

```diff
@@ -20,11 +20,21 @@ export const FilterGroup = styled.div`
 
 // const inactiveBg = `linear-gradient(to top,hsla(355, 43%, 25%, 1),hsla(355, 43%, 17%, 1))`;
 // const activeBg = `linear-gradient(to top, hsla(355, 43%, 50%, 1), hsla(355, 43%, 37%, 1));`;
-const inactiveBg = `linear-gradient(to top,hsla(355, 43%, 17%, 1),hsla(355, 43%, 11%, 1))`;
+const inactiveBg = `linear-gradient(to top, hsla(355, 48%, 38%, 1), hsla(355, 48%, 27%, 1));`;
```

Comment:
It is unclear if these changes are intended for both dark and light mode. The comment below should be deleted or replace the current definition.

Reply:
It's clear now.